src/main/java/io/phdata/streamliner/schemadefiner/ConfigBuilder/ScriptCommand.java

```diff
@@ -0,0 +1,193 @@
+package io.phdata.streamliner.schemadefiner.ConfigBuilder;
+
+import io.phdata.streamliner.schemadefiner.model.*;
+import io.phdata.streamliner.schemadefiner.util.StreamlinerUtil;
+import io.phdata.streamliner.util.JavaHelper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ScriptCommand {
+  private static final Logger log = LoggerFactory.getLogger(ScriptCommand.class);
+
+  public static void build(
+      String configurationFile,
+      String configurationDiffFile,
+      String typeMappingFile,
+      String templateDirectory,
+      String outputDirectory)
+      throws IOException {
+
+    Configuration configuration = StreamlinerUtil.readConfigFromPath(configurationFile);
+    ConfigurationDiff configDiff = StreamlinerUtil.readConfigDiffFromPath(configurationDiffFile);
+    Map<String, Map<String, String>> typeMapping =
+        StreamlinerUtil.readTypeMappingFile(typeMappingFile);
+    if (outputDirectory == null || outputDirectory.equals("")) {
+      outputDirectory =
+          "output/" + configuration.getName() + "/" + configuration.getEnvironment() + "/scripts";
```

Comment:
This would look better with String.format

Reply:
updated.